### ISSUE TITLE
Use admin theme color (1959)

### ISSUE THREAD

**Issue by Mayisha** [woocommerce/woocommerce-paypal-payments#1602]
**Describe the issue**

In a few places, links do not follow the admin theme color. We need to apply the admin theme color there. Use `var(--wp-admin-theme-color)` for color where applicable.

**Toggle button**

<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce-paypal-payments/assets/33387139/937d4e83-532f-4b9f-a901-adc6630f9777" />

**Learn more link**
<img width="50%" height="50%" src="https://github.com/woocommerce/woocommerce-paypal-payments/assets/33387139/3ed35b1e-dae8-465a-a094-fb06a42a13b1" />

**Expected behavior**
The links/texts should follow the admin theme color.

**Comment by InpsydeNiklas** [comment]
Hi @Mayisha 
The plugin generates the clickable `Toggle to manual credential input` link. So the style of this link could be adjusted.
It will be tweaked a bit in the next update:
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70433191/5a075282-79c7-40f3-88f3-4d518804315c)
But we are looking into improving this further and ensuring all styles are consistent.

However, the `Learn more` in the Pay Later messaging is an iframe provided by PayPal. So the only way to style the messaging is by using the available parameters, but we don't have any direct control over the styling.

That being said, the Pay Later messaging configuration will soon be revamped to be easier to use and provide better feedback about what will be displayed on the site. But if you find any other places with inconsistent styling or messaging, please let us know.

**Comment by Mayisha** [comment]
@InpsydeNiklas thanks for the explanation in your previous comment. I went through different settings of WooCommerce again and concluded that we don't need to change the color of the links (the links are rendered in blue in other places as well). So we can ignore the `Toggle to manual credential input` and keep it as it is now.

I found that the Pay Later page has been changed since I opened this issue and the screenshot in the description is not relevant anymore. However, on the Pay Later page, I see a few checkboxes, dropdowns and a toggle that need to follow the admin theme color. 

![before full](https://github.com/woocommerce/woocommerce-paypal-payments/assets/33387139/50d398f0-f70a-4eed-9d53-6285a0d77cbb)

So I have opened a PR with the CSS changes here https://github.com/woocommerce/woocommerce-paypal-payments/pull/2230
